lib/OpenQA/WebAPI/Controller/Running.pm

```diff
@@ -203,25 +204,40 @@ sub streaming {
         Mojo::IOLoop->remove($timer_id);
         $self->finish;
     };
+    my $last_png         = "$basepath/last.png";
```

Comment:
https://circle-production-customer-artifacts.s3.amazonaws.com/picard/forks/58f7029dc9e77c000129905e/56692046/60e88919edca132c54844cbe-0-build/artifacts/artifacts/ui/18-tests-details.t?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Date=20210709T181144Z&X-Amz-SignedHeaders=host&X-Amz-Expires=59&X-Amz-Credential=AKIAJR3Q6CR467H7Z55A%2F20210709%2Fus-east-1%2Fs3%2Faws4_request&X-Amz-Signature=b5ba237bb9977f97ed66b96774deb6e1f620fbd99d53fc1bdc85fcd7db4c59c4
```
[error] [pid:1646] Stopping openqa-webapi process because a Perl warning occurred: Use of uninitialized value $basepath in concatenation (.) or string at /home/squamata/project/lib/OpenQA/WebAPI/Controller/Running.pm line 207.
```